vocabularies/geoadmin-feature-types.ttl

```diff
@@ -0,0 +1,69 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix geof: <http://linked.data.gov.au/def/geofeatures/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/geoadminfeatures#Block> a skos:Concept ;
+    skos:broader <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Block is an administrative division of the State of Queensland that is used for tenure management. A Block is defined as five minutes of latitude by five minutes of longitude. Blocks are grouped by 1:1,000,000 map sheet areas, the names of which provide the containing Blocks with a four letter prefix (e.g. Clermont 1:1,000,000 provides CLER). Each sheet area contains 3456 Blocks, with the Blocks numbered between 1 and 3456 inclusive. A Block is divided into 25 Sub-Blocks."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Block"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#Permit> a skos:Concept ;
+    skos:broader <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "Permits are a license granted over a spatial region within the State of Queensland. Permits govern the activities which can be conducted upon them. Activities include exploration, development, and extraction of a specific commodity or commodities."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Permit"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#SubBlock> a skos:Concept ;
+    skos:broader [ ],
+        <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Sub-Block is an administrative sub-division of a Block. Each Block is divided into 25 Sub-Blocks. A Sub-Block is defined as one minute of latitude by one minute of longitude. The Sub-Blocks are prefixed with their corresponding Block name and are labelled from A to Z, excluding I. Labels are allocated from left to right, top to bottom, starting from the northwest corner."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Sub-Block"@en .
+
+<http://linked.data.gov.au/def/geofeatures> skos:definition "This vocabulary lists types of geological features relevant to the duties of the Geological Survey of Queensland. It is automatically derived from GSQ's Geologic Features Ontology (http://linked.data.gov.au/def/geofeatures)"@en ;
+    skos:prefLabel "Geologic Feature Types"@en .
+
+<https://orcid.org/0000-0002-8742-7730> a owl:NamedIndividual,
```

Comment:
Please remove this section - you are working for GSQ

Reply:
I've removed me